lib/helpers.ts

```diff
@@ -47,22 +47,22 @@ export function postScanner(teamsAndKeywords: TeamAndKeywords) {
     const text = combineText(post); // combine text from post's title, text, and url
     const teamsInterestedInThisPost = new Set() as Set<string>; // set of team IDs that are interested in this post
 
-    text.replace(scanner, (_, ...terms) => {
+    let match;
+    while ((match = scanner.exec(text))) {
```

Comment:
Should this be an equality `==` comparison?

Reply:
No